source/opt/fold.h

```diff
@@ -25,11 +25,13 @@ namespace spvtools {
 namespace opt {
 
 uint32_t FoldScalars(SpvOp opcode,
-                     const std::vector<analysis::Constant*>& operands);
+                     const std::vector<const analysis::Constant*>& operands);
```

Comment:
This needs a method comment out here in the header.    I don't want to have to read the code to know what's going on.